dart/lib/src/protocol/sentry_app.dart

```diff
@@ -48,6 +49,9 @@ class SentryApp {
   /// An app is in foreground when it's visible to the user.
   final bool? inForeground;
 
+  /// The current application screen.
+  final String? screen;
```

Comment:
Maybe we call something more generic like `view_name` or something, so it's more generic?
It'd be cool to give a few examples as well, for example, on Android this is the Acitivty name.
This has to be added here as well https://github.com/getsentry/relay/blob/master/relay-general/src/protocol/contexts/app.rs

Reply:
I made a fork and added a draft PR: https://github.com/getsentry/relay/pull/2344

Reply:
The PR over at relay is approved. I updated title/description there as suggested.

Reply:
And it's merged. 🥳